vdyp-common/src/main/java/ca/bc/gov/nrs/vdyp/application/RuntimeProcessingException.java

```diff
@@ -0,0 +1,16 @@
+package ca.bc.gov.nrs.vdyp.application;
```

Comment:
IMO using RuntimeExceptions for application level errors is bad practice. Why is this needed?